[DM-710] updated user documentation of Actility and Sigfox device registration (#533)
SoftwareAG/c8y-docs: content/protocol-integration/sigfox-bundle/sigfox-troubleshooting.md

#### No Sigfox provider settings are found

> This warning message shows up when there are no credentials set up for the Sigfox account. To resolve this click on **Settings** to navigate to administration app where credentials are configured.

Reviewer: ```suggestion
This warning message shows up when there are no credentials set up for the Sigfox account. To resolve this click **Settings** to navigate to the Administration application where the credentials are configured.
```

Author: Corrected.

> This warning message shows up when there are no credentials set up for the Sigfox account. To resolve this click **Settings** to navigate to the Administration application where the credentials are configured.

Author: Corrected.

<img src="/images/device-protocols/sigfox/sigfox-troubleshooting-registration.png" alt="Device registration failure without credentials" style="max-width: 100%">
<br>

#### No device protocols configured

> This warning message shows up when no Sigfox device protocol exists to be used for device registration. To resolve this, click on **Device protocols** to navigate to device protocols page where protocols are configured.

Reviewer: ```suggestion
This warning message shows up when no Sigfox device protocol exists to be used for device registration. To resolve this, click  **Device protocols** to navigate to the **Device protocols** page where the protocols are configured.
```

Author: Corrected.

> This warning message shows up when no Sigfox device protocol exists to be used for device registration. To resolve this, click  **Device protocols** to navigate to the **Device protocols** page where the protocols are configured.

Reviewer: ```suggestion
This warning message shows up when no Sigfox device protocol exists to be used for device registration. To resolve this, click **Device protocols** to navigate to the **Device protocols** page where the protocols are configured.
```
